src/server/api/controllers/users.controller.js

```diff
@@ -6,35 +6,50 @@ const createUser = async (body) => {
   await knex('users').insert({
     full_name: body.full_name,
     email: body.email,
+    mobile: body.mobile,
     address: body.address,
     zipcode: body.zipcode,
     city: body.city,
     country: body.country,
     created_at: moment(Date.now()).format(),
   });
-
   return {
     successful: true,
   };
 };
-
-// eslint-disable-next-line
+const getUserById = async (user_id) => {
+  const id = parseInt(user_id, 10);
+  if (isNaN(id) || id < 1) {
```

Comment:
@miltonhultgren isNan(id) returning "3gt" as "3" and showing result for product "3", 
Why can not we use the isNaN(user_id) instead of isNaN(id) ?